packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -15,6 +15,9 @@ interface Input {
 
 export default class MeetingTeamPrompt extends Meeting {
   meetingType!: 'teamPrompt'
+  commentCount?: number
+  taskCount?: number
+  responseCount?: number
```

Comment:
-2 You defined these fields in two places, the meeting and the timeline event. I think having this data static in meeting is misleading and may cause bugs later. In meeting I would expect those to be live updated. In timeline event though I'm fine with having a snapshot in time.

Reply:
👍 changed to dynamic updating